AmmadAmjad-CodeTest/Server/controllers/db.js

```diff
@@ -0,0 +1,49 @@
+const Sequelize = require('sequelize');
```

Comment:
Not sure if `controllers` is a good place to put `db.js`, db can be in a separate place where all "connections" related stuff / external dependencies stuff / data access stuff live together.

`db-test.sqlite` and `db.sqlite` don't seem to be necessary to commit to the repo. If you want to seed the server, you can make a seed SQL script.

----

Just realized you are using `controllers` folder as the purpose I mentioned, so it's OK, as long as it's consistent.